linode/manager
# M3-5596: Convert ActionMenu story to MDX

## Description
- Converts `ActionMenu.stories.tsx` to MDX
- Removes `inlineLabel` prop from `ActionMenu.tsx` and removes use of said prop from `LinodeActionMenu.tsx`

## How to test
Run `yarn storybook` and check for "ActionMenu" in the `Components` section. Check both the "Canvas" and "Docs" tabs.

Also check instances of action menus in Cloud to confirm there have been no regressions.
